src/renderer/certificate/lens-proxy-certificate.injectable.ts

```diff
@@ -26,10 +22,6 @@ const lensProxyCertificateInjectable = getInjectable({
         certState = certificate;
```

Comment:
Probably should also verify that set is not called multiple times

Reply:
Good point... and we can probably have just a single implementation under "common" now.

Reply:
Yes

Reply:
Done.